Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi add-ons: video/plugin.video.youtube](https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/).



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widget/kodi-add-ons-video/plugin-video-youtube/horizontal-auto.svg)

MoojMidge commented:
@mapi68 - thanks for the update to the translation. Question though - is there a reason why a space is needed for these now unused strings?

mapi68 commented:
@MoojMidge Otherwhise Weblate doesn't validate translate

MoojMidge commented:
I see. I'll have to check whether I should just be deleting these instead

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (388 of 388 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-video/plugin-video-youtube/it_it/
Translation: Kodi add-ons: video/plugin.video.youtube
